clients/drcachesim/tests/raw2trace_unit_tests.cpp

```diff
@@ -493,6 +512,23 @@ test_branch_delays(void *drcontext)
         check_entry(entries, idx, TRACE_TYPE_INSTR_DIRECT_JUMP, -1) &&
         check_entry(entries, idx, TRACE_TYPE_ENCODING, -1) &&
         check_entry(entries, idx, TRACE_TYPE_INSTR, -1) &&
+        check_entry(entries, idx, TRACE_TYPE_ENCODING, -1) &&
+#ifdef X64
+        // TODO i#7469: This is a bug. When looking for the kernel_event marker,
```

Comment:
#7469 is this pull request.  I assume you mean something else, a filed issue?  If you meant something other than #7470 please add that issue # to the PR description; if you meant #7470 please change the `Fixes` in the PR description.

Reply:
Yes. Changed both places.